src/app/services/pages.service.spec.ts

```diff
@@ -1,18 +1,32 @@
 import { TestBed } from '@angular/core/testing';
-
 import { PagesService } from './pages.service';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
 describe('PagesService', () => {
-  beforeEach(() => TestBed.configureTestingModule(
-    {
-    }));
+  let pagesService: PagesService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({imports:[BrowserAnimationsModule]});
+    pagesService = TestBed.get(PagesService);
+  });
 
   it('should be created', () => {
-    const service: PagesService = TestBed.get(PagesService);
+    expect(pagesService).toBeTruthy();
+  });
 
-    expect(service).toBeTruthy();
-  
+  it('test constructor', () => {
+    pagesService = new PagesService();
+    expect(pagesService).toBeDefined();
   });
 
+  // Test only serves the purpose of full coverage since we are unable to spy on window since it is global.
+  // Also normal attmpt to test window.location.reload ends in loop.
```

Comment:
Perhaps more information can be provided about this. Future associates may need to fix such issues in a different sprint. More information about issues is always helpful.

Reply:
Perhaps we can document this on asana?